src/pip/_internal/cli/req_command.py

```diff
@@ -184,19 +183,18 @@ def warn_if_run_as_root():
     )
 
 
-def with_cleanup(func):
-    # type: (Any) -> Any
+def with_cleanup(func: Any) -> Any:
```

Comment:
This annotation is pretty lazy and could definitely use some improvements, but that’s a separate issue I guess.

Reply:
Ok. Maybe on the next week (it's Sunday here) I can take a look on that.